src/Mindee/Input/MindeeInputUtils.cs

```diff
@@ -0,0 +1,226 @@
+using System;
+using System.Runtime.InteropServices;
+using Docnet.Core.Models;
+using Mindee.Exceptions;
+using SkiaSharp;
+
+namespace Mindee.Input
+{
+    /// <summary>
+    /// Utility class.
+    /// </summary>
+    public static class MindeeInputUtils
```

Comment:
these are not only for input, suggest putting in a ImageUtils class

Reply:
I was thinking the same initially, but wouldn't that mean that they'd belong to the Geometry namespace? Because I kind of do not want to move it there :grimacing:

Reply:
I'll leave it in Input for now since about 90% of the content is used there exclusively.